src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -335,51 +344,244 @@ private synchronized void writeToConsole(Person person, int index, long time, bo
+		  String ethnicity = city.ethnicityFromRace((String)person.attributes.get(Person.RACE), person);
+		  person.attributes.put(Person.ETHNICITY, ethnicity);
+	  }
+ 	  
+	  String language = city.languageFromEthnicity((String) person.attributes.get(Person.ETHNICITY),person);
+	  person.attributes.put(Person.FIRST_LANGUAGE, language);
+
+	  String gender = spewPerson.get(rand_spew).get("SEX");
+
+	  if(gender.equals("1")){
+		  person.attributes.put(Person.GENDER, "M");
+	  } else if (gender.equals("2")){
+		  person.attributes.put(Person.GENDER, "F");
+	  }
+
+	  //Longitude and latitude
+	  //TODO a look up to assign address/city/town/zip from lat and long
+	  //look into using FIPS codes
+	
+	  String longitude = spewPerson.get(rand_spew).get("longitude");
+	  person.attributes.put(Person.LONGITUDE, longitude);
+	
+	  String latitude = spewPerson.get(rand_spew).get("latitude");
+	  person.attributes.put(Person.LATITUDE, latitude);
+
+	  String nativity = spewPerson.get(rand_spew).get("NATIVITY");
+	
+	  if(nativity.equals("1")) {
+		  person.attributes.put(Person.NATIVITY, "native");
+	  } else if(nativity.equals("2")) {
+		  person.attributes.put(Person.NATIVITY, "foreign_born");
+	  }
+	
+	  List<LinkedHashMap<String, String>> birthplaces;
+	  
+	  try {
+		  birthplaces = SimpleCSV.parse(Utilities.readResource("birthplaces.csv"));
+	  } catch (IOException e) {
+		  e.printStackTrace();
+		  return (Long) null;
+	  }
+	    
+	  for(int i = 0;i<=birthplaces.size()-1;i++) {
+		  if(spewPerson.get(rand_spew).get("POBP").equals(birthplaces.get(i).get("Code"))) {
+			  person.attributes.put(Person.BIRTHPLACE, birthplaces.get(i).get("Pob"));
+		  } 
+	  }
+	  
+	  String school_enrollment = spewPerson.get(rand_spew).get("SCH");
+
+	  if(school_enrollment.equals("NA")){
+		  person.attributes.put(Person.SCHOOL_ENROLLMENT, "N/A (less than 3 years old)");
+	  } else if (school_enrollment.equals("1")){
+		  person.attributes.put(Person.SCHOOL_ENROLLMENT, "no");
+	  } else if (school_enrollment.equals("2")){
+		  person.attributes.put(Person.SCHOOL_ENROLLMENT, "public_school_or_public_college");
+	  } else if (school_enrollment.equals("3")){
+		  person.attributes.put(Person.SCHOOL_ENROLLMENT, "private_school_or_college_or_home_school");
+	  } 
+	  
+	  List<LinkedHashMap<String, String>> grade_level;
+	  
+	  try {
+		  grade_level = SimpleCSV.parse(Utilities.readResource("grade_level.csv"));
+	  } catch (IOException e) {
+		  e.printStackTrace();
+		  return (Long) null;
+	  }
+	    
+	  for(int i = 0;i<=grade_level.size()-1;i++) {
+		  if(spewPerson.get(rand_spew).get("SCHG").equals(grade_level.get(i).get("Code"))) {
+			  person.attributes.put(Person.GRADE_LEVEL, grade_level.get(i).get("grade"));
+		  } 
+	  }
+	  
+	  List<LinkedHashMap<String, String>> relationship;
+	  
+	  try {
+		  relationship = SimpleCSV.parse(Utilities.readResource("relationship.csv"));
+	  } catch (IOException e) {
+		  e.printStackTrace();
+		  return (Long) null;
```

Comment:
Again. Also, reading these files in for each person doesn't make a whole lot of sense performance wise. You should be caching these files.